cypress/pages/LoginPage.js

```diff
@@ -0,0 +1,24 @@
+class LoginPage{
+
+    emptyLoginFieldMessageCheck(){
+        cy.xpath(`//*[text()='Pole Login jest wymagane.']`).should('contain.text', 'Pole Login jest wymagane.')
```

Comment:
tu raczej pytanie niz request change - nie dalo sie tego zrobic bez xpatha?
generalnie xpathy to ostatecznosc bo sa o wiele wolniejsze niz dom selektory

Reply:
wiesz co poki co nie udalo mi sie tego inaczej wylapac niz xpathem